ROADMAP.md

```diff
@@ -21,40 +27,35 @@ and interactive context, along with relationship discovery.
 
 - [x] Migration to the [Trait based API](https://github.com/OpenAssetIO/OpenAssetIO/blob/main/doc/decisions/DR007-Hierarchical-or-compositional-traits-for-specifications.md).
 - [x] Migration to batch-centric API.
-- [ ] Migration of core API methods to C+. [In progress]
-- [ ] Sundry breaking changes and tech-debt cleanup.
+- [x] Migration of core API methods to C+.
+- [ ] Sundry breaking changes and tech-debt cleanup. [In progress]
 
 ## Y24 Q1: v1.0.0
 
+**Backlog:** [here](https://github.com/OpenAssetIO/OpenAssetIO/milestone/7)
+
 This release will provide additional introspection methods and
 debugging functionality, along with language parity between Hosts and
 Managers.
 
 **Hosts:** C++, Python
 **Managers:** C++, Python
 
-- [ ] Auth related error codes.
 - [ ] Debug trace logging support.
 - [ ] Entity introspection API methods.
 - [ ] C++ Plugin System
 - [ ] Hybrid C++/Python manager bridge.
-- [ ] Out-of-process Python
+- [ ] Read-through cache mix-ins.
```

Comment:
[question] Are these cache mixins distinct from the "Out-the-box caching mechanisms for resolve" listed under Y24 onwards?

Reply:
See https://github.com/OpenAssetIO/OpenAssetIO/pull/1116#discussion_r1332874530